components/get_android_app/get_android_app.jsx

```diff
@@ -59,12 +58,16 @@ export default function GetAndroidApp({androidAppDownloadLink}) {
                     defaultMessage='Or {link}'
                     values={{
                         link: (
-                            <Link to='/'>
+                            <a
+                                onClick={history.goBack}
```

Comment:
I'm hesitant to rely on `history.goBack()` -- while it likely works now due to a bunch of redirects, if we ever allowed the `/get_ios_app` to be accessed directly (e.g. from a link outside the server), going back would be the wrong action to take.

I think it would generally be more robust if we modified the redirection to code in `root.jsx` to add a `redirect_to` querystring parameter to which the user should redirect afterwards, and have it default to `/` here when not provided for any reason.

Reply:
For sure, redirect approach is much better. Using `history.goBack()` was giving off a strong smell.